.azure-pipelines-preproduction/package-lock.json

```diff
@@ -24,9 +24,9 @@
       }
     },
     "@azure/oad": {
-      "version": "0.8.0",
-      "resolved": "https://registry.npmjs.org/@azure/oad/-/oad-0.8.0.tgz",
-      "integrity": "sha512-RuPG1JP4lcDIcQiFQOBWq/P6L9FUx42A7zGyTsEOPglJdE3CE5h2NseAzT3k3NlO4ze+1jTn5eKyX3ER+K3PRA==",
+      "version": "0.8.1",
+      "resolved": "https://registry.npmjs.org/@azure/oad/-/oad-0.8.1.tgz",
+      "integrity": "sha512-Xl5vTMkfPkv0wVUgbvfITpaV07o6RIPOWpyPcCiZgQpkpfc0vsSh4SK3K4JSJ44z4BXTOhCSmkYgv/Zvtq/vSQ==",
```

Comment:
Should we also change package.json?

Reply:
Don’t need,the oad version is locked by package-lock.json and The oad does not exist in package.json  because it is installed as a dependency